src/DeviceInterface/tests/password-feature.test.js

```diff
@@ -98,24 +100,18 @@ describe('AppleDeviceInterface: postAutofill', () => {
         spy = pmHandlerStoreDataSpy()
     })
     it('performs a save if a generated password was used', async () => {
+        const formEl = attachAndReturnGenericForm()
         const device = createDevice()
+        const scanner = createScanner(device).findEligibleInputs(document)
+        const formClass = /** @type import('../../Form/Form.js').Form */(scanner.forms.get(formEl))
```

Comment:
As mentioned above, now we need to create the form here. Fortunately, `attachAndReturnGenericForm` makes this rather easy.